src/protocols/bitcoin/libs/BitcoinAdapter.ts

```diff
@@ -47,10 +59,13 @@ export class BitcoinAdapter extends BaseProtocolAdapter {
     };
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   override async getBalance(address: string): Promise<string> {
```

Comment:
By the way: what do you think about renaming this to `fetchBalance`? This would be more close to our agreements we did few months ago.

Reply:
Done. Thanks